backend/routes/programs.js

```diff
@@ -25,28 +23,73 @@ router.get("/all", function(req, res) {
     });
 });
 
+// Create a program
 router.post("/", function(req, res) {
-  db["Authentication"].programs
-    .updateOne(
-      {
-        databaseName: req.body.databaseName,
-        displayName: req.body.displayName
-      },
-      req.body,
-      { upsert: true }
-    )
-    // status code 201 means created
-    .then(function(newSchedule) {
-      //This is a helper function in mongo.js that adds another connection
-      addConnection(req.body.databaseName);
-      res.status(201).json(newSchedule);
-    })
-    .catch(function(err) {
-      res.send(err);
-    });
+  try {
+    // TODO: call programsServices.createProgram()
+    res.status(201).send();
+  } catch (e) {
+    res.send(e);
+  }
```

Comment:
Just a placeholder for now, this should be implemented in a future PR when the ticket becomes unblocked